martus-common/source/org/martus/common/test/TestSha1ToSha2Change.java

```diff
@@ -252,6 +254,57 @@ private File createAccountMapAndSignature(MartusCrypto security) throws Exceptio
 		return dir;
 	}
 
+	public void testServerFileDbInitializerWhenAccountMapSignedWithSha1UsingSha2Verifier() throws Exception
+	{
+		File dir = createServerAccountMapAndSignature(securitySha1);
+
+		try
+		{
+			ServerFileDatabase cfdb = new ServerFileDatabase(dir, securitySha2);
+			cfdb.initialize();
```

Comment:
Should include a assert to show the initialize passed.  Have a catch exception that fails with a message that initialize should have successful without exception